relay-dynamic-config/src/defaults.rs

```diff
@@ -143,6 +147,63 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
                 })
                 .into(),
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "c:spans/data.llm.llm_completion_tkens@none".into(),
+            field: Some("span.data.llm_completion_tkens".into()),
+            condition: Some(RuleCondition::eq("span.op", "ai.llm.completion")),
+            tags: [
+                ("span.sentry_tags.", "group"),
+                ("span.sentry_tags.", "op"),
+                ("span.data.", "language_model"),
+                ("span.sentry_tags.", "environment"),
```

Comment:
This would end up as tag `span.environment`, but we already have a precedent for naming that tag `environment`. Probably easiest to list `TagSpec` objects explicitly here instead of using `.map()`.